apps/site/src/app/(home)/sections/Sponsors/Sponsors.tsx

```diff
@@ -3,31 +3,35 @@ import { getSponsors } from "./getSponsors";
 import styles from "./Sponsors.module.scss";
 import { client } from "@/lib/sanity/client";
 import imageUrlBuilder from "@sanity/image-url";
-import clip from "./clip.svg";
+import Container from "react-bootstrap/Container";
+import Clubs from "../Clubs/Clubs";
 
 const builder = imageUrlBuilder(client);
 
 const Sponsors = async () => {
 	const sponsors = await getSponsors();
 
 	return (
-		<section className={styles.container}>
-			<div className={styles.clipboard}>
-				<img className={styles.clip} src={clip.src} alt="" />
-				<h2 className={styles.title}>Sponsors</h2>
-				<div className={styles.logos}>
-					{sponsors.sponsors.map(({ _key, name, url, logo }) => (
-						<a key={_key} href={url} target="_blank" rel="noopener noreferrer">
-							<img
-								className={styles.logo}
-								src={builder.image(logo).format("webp").url()}
-								alt={name + " logo"}
-							/>
-						</a>
-					))}
-				</div>
+		<Container as="section">
+			<h2 className={styles.title}>SPONSORS</h2>
+			<div className={styles.logos}>
+				{sponsors.sponsors.map(({ _key, name, url, logo, tier }) => (
+					<a
+						key={_key}
+						href={url}
+						target="_blank"
+						rel="noopener noreferrer"
+						className={`${styles.logo} ${styles[tier]}`}
+					>
+						<img
+							src={builder.image(logo).format("webp").url()}
```

Comment:
Thought: to keep the assets size smaller, you can add `.width(500)` to keep the width of the assets fetched 500px wide.